packages/modules/web-viewer-react/src/services/Initializer.ts

```diff
@@ -85,20 +87,27 @@ export class WebInitializer {
       console.log("starting web viewer");
       this._initializing = true;
       const cancellable = makeCancellable(function* () {
+        ViewerPerformance.enable(options?.enablePerformanceMonitors);
```

Comment:
options is required, as of the auth change pr, can drop the ?.

Reply:
fixed